deploy/helm/scf/assets/operations/instance_groups/log-api.yaml

```diff
@@ -48,3 +48,12 @@
     run:
       memory: 128
       virtual-cpus: 2
+
+- type: replace
```

Comment:
We don't need this `replace`. We already have a `replace` adding `quarks` (you will need to rebase with v3-develop where it is renamed from `bosh_containerization` to `quarks`) on line 42. You will just need to add:
```
    envs:
    - name: TRAFFIC_CONTROLLER_IP
      valueFrom:
        fieldRef:
          apiVersion: v1
          fieldPath: status.podIP
```